mautic/mautic
# Add support for list inserted/updated contacts in import

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| Automated tests included? | yes
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | no
| Deprecations? | no 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new clickable columns in Import history to display contacts related to status of each import (Inserted/Updated). 

![image](https://user-images.githubusercontent.com/462477/51744240-736e2d00-209f-11e9-9226-d415c5652bba.png)

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com/7171)
2. Import CSV Note: mock data can be generated at https://www.mockaroo.com/ very quickly/easily to test this PR!
3. Go to Import history
4. Click on columns and see If contacts display correctly based on import status
